CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Datatrans Changelog
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## 4.0.0 - 2022-02-25
+---
```

Comment:
Those lines aren't necessary anymore, they're just a different way to write headings in markdown.

Reply:
yep, you're right